### PR TITLE
Creating the new Internet2 caches registration

### PR DESCRIPTION
**Pull request by biozit**
We are installing 5 new OSDF caches on I2; I am creating all the configurations in advance to deploy the cache as soon as possible.

### CODE REVIEW

**Comment by brianhlin** [comment]
@biozit do you have a relevant ticket? A descriptive PR title + commit and a description would go a long way to helping us understand the PR!

**Comment by biozit** [comment]
This is the registration of the new I2 OSDF caches, @djw8605 is the PI.

File: topology/Internet2/Internet2Boise/Site.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ID: 10348
+LongName: InternetBoise
+Description: Backbone location of Internet2 at the Boise POP
```
Comment (brianhlin) [inline topology/Internet2/Internet2Boise/Site.yaml:3]:
```suggestion
LongName: Internet2 Boise
Description: Backbone location of Internet2 at the Boise Point of Presence
```

File: topology/Internet2/Internet2Jacksonville/Site.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+ID: 10349
+LongName: InternetJacksonville
+Description: Backbone location of Internet2 at the Jacksonville POP
```
Comment (brianhlin) [inline topology/Internet2/Internet2Jacksonville/Site.yaml:3]:
```suggestion
LongName: Internet2 Jacksonville
Description: Backbone location of Internet2 at the Jacksonville Point of Presence
```

**Review by brianhlin — CHANGES_REQUESTED** [review]
Some minor changes to the LongName and Descriptions for the sites (we should spell out acronyms in descriptions/long names wherever possible). The CI is complaining about missing `SITE.yaml` so I think that the filenames are case sensitive.

Do you have DNs? We may also have to update the IDs

**Comment by biozit** [comment]
I requested the cert to I2

**Comment by biozit** [comment]
@brianhlin could we merge Boise and Cincinnati? these two caches are ready!

**Comment by brianhlin** [comment]
Hi @biozit this PR has more caches than just Boise and Cincinnati. Can you split those changes off into separate PRs?

**Comment by biozit** [comment]
https://github.com/opensciencegrid/topology/pull/2880

**Comment by edquist** [comment]
Hi @biozit, I'm seeing the following error in our checks:
```
Missing site Internet2Jacksonville for RG Site.  Is there a SITE.yaml?
```

**Comment by biozit** [comment]
Hi @edquist, yes it is on the PR.

**Comment by brianhlin** [comment]
Hey @biozit , is this ready for review? It's been a bit of a long-lived PR that seems to keep getting caches added to it and it may get a bit hairy when merging `master` back in and resolving conflicts. I'd suggest making a PR per cache (or batching a handful of caches together if they're all complete). We can help with cherry picking commits to different branches.

As for the issue with the missing site, it looks like our check is for `SITE.yaml` and is case sensitive =/.

**Comment by biozit** [comment]
@brianhlin this PR is ready to review.

**Comment by bbockelm** [comment]
Looks like lat/long is missing for Denver.

Please rebase with more descriptive commits (or squash if necessary)

**Comment by biozit** [comment]
done

**Comment by brianhlin** [comment]
@biozit did you see Brian's request here?

> Please rebase with more descriptive commits (or squash if necessary)

Looking at the history, it doesn't look like like the commits have been rebased/reworded/squashed. If you haven't worked with `git rebase` before, I like these articles https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase and https://www.atlassian.com/git/tutorials/merging-vs-rebasing.

As for for rewording or squashing the commits themselves, I'd read through https://osg-htc.org/technology/software/git-software-development/#making-good-pull-requests-the-art-of-good-commits

**Comment by biozit** [comment]
Let me create a new PR; this one is too fuzzy

**Pull request closed without merging**